pkg/connectorrunner/runner.go

```diff
@@ -45,20 +45,35 @@ var ErrSigTerm = errors.New("context cancelled by process shutdown")
 
 // Run starts a connector and creates a new C1Z file.
 func (c *connectorRunner) Run(ctx context.Context) error {
+	l := ctxzap.Extract(ctx)
 	ctx, cancel := context.WithCancelCause(ctx)
 	defer cancel(ErrSigTerm)
 
 	if c.tasks.ShouldDebug() && c.debugFile == nil {
 		var err error
-		c.debugFile, err = os.Create(filepath.Join(c.tasks.GetTempDir(), "debug.log"))
+		tempDir := c.tasks.GetTempDir()
+		if tempDir == "" {
```

Comment:
how about just

```_, err := os.Stat(tempDir)
	if err != nil {
		if errors.Is(err, os.ErrNotExist) {
			l.Warn("debug log file does not exists", zap.Error(err))
			return nil
		}
		return err
	} else {
	... the rest of the code
```
that's what what we do here https://github.com/ConductorOne/baton-sdk/blob/main/pkg/tasks/c1api/full_sync.go#L190

Reply:
I've have changed this and the other file (full_sync.go) too. Thoughts?

Reply:
as in having consistent code across files? Well, that could work too